src/feature/flow_chart_panel/views/ControlBar.tsx

```diff
@@ -80,9 +82,19 @@ const ControlBar = () => {
       <div className="flex">
         <Menubar>
           <MenubarMenu>
-            <MenubarTrigger id="file-btn" data-testid="file-button">
-              File
-            </MenubarTrigger>
+            <DemoWarningTooltip
+              tooltipContent={
+                "Download the desktop app to save and load flowcharts!"
+              }
+            >
+              <MenubarTrigger
```

Comment:
Would be nice to have some different "disabled" style for this, it doesn't look like it's disabled until you hover over it, probably grey out the text (?)